examples/credential_offer_pre-authz_code.json

```
@@ -7,7 +7,7 @@
         "urn:ietf:params:oauth:grant-type:pre-authorized_code": {
             "pre-authorized_code": "adhjhdjajkdkhjhdj",
             "tx_code" : {
-               "description" : "Enter the PIN which was send by SMS to your mobile phone"
+               "description" : "Please provide the transcation code which was sent to your mobile phone via SMS"
```

I think `one-time code` is he way to go. we already established in PR #159 that `transaction code` is not understood by the user.

```suggestion
               "description" : "Please provide the one-time code which was sent to your mobile phone via SMS"
```

---

ok sorry, I wasn't aware of the consensus on the term "one-time code" for the description. I just found it strange to introduce another new term in this example. I would prefer to stick with term "transcation code" or give a clear recommendation for the term "one-time code".